Flipper/Packages/UI/Sources/Common/Image/SVG/SVGView.swift

```diff
@@ -0,0 +1,80 @@
+import SwiftUI
+
+struct SVGView: View {
+    let commands: [SVGCommand]
+
+    var body: some View {
+        Path { path in
```

Comment:
I'm not sure if swiftui can cache such views and wont render each image for dozens of apps